cmd/amplifier/main.go

```diff
@@ -52,6 +56,8 @@ func parseFlags() {
 	flag.StringVarP(&clientSecret, "clientsecret", "c", defaultClientSecret, "github app clientsecret (default '"+defaultClientSecret+"')")
 	flag.StringVarP(&kafkaURL, "kafkaURL", "k", kafkaDefaultURL, "kafka URL (default '"+kafkaDefaultURL+"')")
 	flag.StringVarP(&influxURL, "influxURL", "", influxDefaultURL, "InfluxDB URL (default '"+influxDefaultURL+"')")
+	flag.StringVarP(&dockerURL, "dockerURL", "", dockerDefaultURL, "Docker URL (default '"+dockerDefaultURL+"')")
```

Comment:
Don't use `*P` (`flag.StringVarP`) if you're not going to provide a short parameter and remove the corresponding empty `""` argument. Same with the next option (and also the same for the previous one, although you didn't do it.

Also, don't create options for linux with mixed case; that's not standard. Only use lowercase (and hyphens, if necessary). Ex: `--docker-version`.  However, I don't understand why we need to provide these options anyway. We only build and test for a specific version, so does it make sense to make the version an option? The version needs to be coordinated with the client. Also, why do we need docker url (will it ever be at a location other than `unix://var/run/docker.sock`)?